documentation/configuration.markdown

```diff
@@ -144,8 +144,13 @@ The init configuration file location depends on your operating system.
     ```
     Jun 01 19:58:16 myhost puppetserver[24001]: Unable to find specified JRUBY_JAR:/opt/puppetlabs/server/apps/puppetserver/jruby-wrong.jar
     ```
+2.  Set other configuration options for better performance
 
-2.  Restart the `puppetserver` service.
+    To ensure good performance with JRuby 9k, the following settings should also be updated:
+    * In `puppetserver.conf`, `jruby-puppet.compile-mode` should be set to `jit`. This setting must be switched back when reverting to 1.7.
```

Comment:
In Server 5.3.0 and later, if using JRuby 9k, `jit` is the default compile mode and doesn't need to be toggled.

Reply:
We can update this when we merge up then to leave that part out.

Reply:
This section doesn't even exist at all anymore in the master branch, so that is already taken care of.

Reply:
Master's 6.0.0 now and 9k only, right?

Reply:
Yep.